src/main/java/core/basesyntax/strategy/impl/PurchaseHandler.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.Map;
+
+public class PurchaseHandler implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction fruitTransaction) {
+        int quantityToBuy = fruitTransaction.getQuantity();
+        String fruit = fruitTransaction.getFruit();
+        Map<String, Integer> storageMap = Storage.storageMap;
```

Comment:
I think this storageMap local variable is redundant. Please fix it in all places